ArcGISRuntimeSDKQt_CppSamples/Maps/TakeScreenshot/main.cpp

```diff
@@ -54,7 +54,7 @@ int main(int argc, char *argv[])
 
   // Add the import Path
   view.engine()->addImportPath(QDir(QCoreApplication::applicationDirPath()).filePath("qml"));
-  // Add the Runtime and Extras path
+  // Add the Runtime path
   view.engine()->addImportPath(arcGISRuntimeImportPath);
   // Add the Toolkit path
   view.engine()->addImportPath(arcGISToolkitImportPath);  
```

Comment:
I wonder if we can also review requirements for the toolkit import for the C++ samples? I guess it's only those that use the `AuthenticationView`, `Callout` etc. that would need it?

Reply:
yeah, probably a good idea. i'll try and do this, but not as part of this PR

Reply:
it's probably not super critical, because it isn't really requiring or including anything - it is just saying to additionally scan this location for QML plugin imports. Our project/pri files already don't actually include or bring in the toolkit - they just set up the import paths so that if one did want to import the toolkit, it would "just work"